acmw-db-app/components/AttendanceForm.jsx

```diff
@@ -171,9 +173,24 @@ const SubmitButton = (props) => {
       } else if (!firstnameCheck) {
         setMessage("Please enter your first name.");
       }
-    }
+    }*/
   } 
 
+  const recordAttendance = async () => {
+        const requestAttendanceRecord= {
+          method: 'POST',
+          body: JSON.stringify(
+            { event_code: props.eventCode,
+              f_name: props.firstName.charAt(0).toUpperCase() + props.firstName.slice(1).toLowerCase(), //capitalization convention
```

Comment:
I think it would be better to not change capitalization since there's names like "Sarah Ann" or "D'Shawn"

Reply:
ok i was following convention in another file but I will change

Reply:
ope which file

Reply:
the AddAccountForm.jsx but if it doesnt apply here nbd

Reply:
oh oopsies should i get rid of the capitalisation in `AddAccountForm`